src/src/CustomBuild/define_plugin_sets.h

```diff
@@ -2918,7 +2918,7 @@ To create/register a plugin, you have to :
 // Extra task value types, typically used in Dummy tasks.
 // For example 32-bit, 64-bit ints and doubles.
 #ifndef FEATURE_EXTENDED_TASK_VALUE_TYPES
-  #ifdef ESP8266_1M
+  #ifdef LIMIT_BUILD_SIZE
     #define FEATURE_EXTENDED_TASK_VALUE_TYPES  0
```

Comment:
This seems quite limiting in features for all Collection builds?

Reply:
That's the extra added dummy types, like uint64, double.

Reply:
The display ESP8266 build:
```
Flash: [==========]  99.2% (used 1035776 bytes from 1044464 bytes)
```
vs
```
Flash: [==========]  99.3% (used 1037012 bytes from 1044464 bytes)
```